hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestMergeIntoTable.scala

```diff
@@ -115,6 +116,65 @@ class TestMergeIntoTable extends HoodieSparkSqlTestBase with ScalaAssertionSuppo
     })
   }
 
+  test("Test MergeInto with more than once update actions") {
+    withRecordType()(withTempDir {tmp =>
+      val conf = new SparkConf().setAppName("insertDatasToHudi").setMaster("local[*]")
+      val spark = SparkSession.builder().config(conf)
+        .config("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+        .config("spark.sql.extensions", "org.apache.spark.sql.hudi.HoodieSparkSessionExtension")
```

Comment:
these config can be removed and please refer to other tests.

```
      val conf = new SparkConf().setAppName("insertDatasToHudi").setMaster("local[*]")
      val spark = SparkSession.builder().config(conf)
        .config("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
        .config("spark.sql.extensions", "org.apache.spark.sql.hudi.HoodieSparkSessionExtension")
```

Reply:
thanks，temple view need HoodieSparkSessionExtension，so i add this

Reply:
other tests also set HoodieSparkSessionExtension.

Reply:
> other tests also set HoodieSparkSessionExtension.

yes，i had change it